fb_veri.py

```diff
@@ -1,14 +1,35 @@
+'use strict';
 
-def handler(event, context):
-    # Dummy verification lambda, just replies 200 OK every time.
-    # Will need to pull the verification token and challenge string
-    #  out of event['queryStringParameters'] and verify the first, return
-    #  the second.
-    response = {
-        'statusCode': 200,
-        'body': '',
-        'headers': {
-            'Content-Type': '*/*'
-        }
+#var USER_TOKEN = "EAACuOJLUNcoBAL8PoBGXgDVrKvv05oPsvclS0YDavfozZAl86WcHkYTM5kcalUQkt3ZBoPwjfcr07Rryf8bFfzqnjZB71erjR4gSK2tswNPOGbNmT7HrhCUe0PDvRQx864sMl9849jk8gN7C0pJff9Hx9QgYTIZD";
+var VERIFY_TOKEN = "Access_Token";
+
+def handler (event, context):
+ 
+  if(event['queryStringParameters']){
```

Comment:
Remove the parentheses in the if and the curly braces for everything that isn't a dictionary (everything but lines 16, 19, 23, 25, 30, 32)